WME_Image_Overlays.user.js

```diff
@@ -77,12 +53,7 @@ function init(e) {
   setTranslations(io_strings);
   applyStyles();
 
-  var tab = addTab();
-
-  // Deal with measuring unit switches between metric and imperial
-  W.prefs.on('change:isImperial', function() {
-    addTab(tab);
-  });
+  var tab = await addTab();
```

Comment:
Changed to an await. Otherwise, tab is settled as undefined before addTab() is actually complete. Causing the rest of this function to error when referencing {tab}.